### DISCUSSION TITLE
New server Submission

### DISCUSSION THREAD

**Discussion by ehsan0921** [mastodon/mastodon#21847] [General]
Hi all
I have two instances and I can't submit it to the Mastodon server list.
I email them but no response yet.
Any suggestions?

**Comment by MrGeneration** [comment]
Patience is all you can do.
If they don't reply it's unlucky.

I haven't received feedback in two weeks.
Possibly they aren't answering to mails of instances that don't apply or they're just overloaded. Who knows.

**Comment by Nobodysbetter** [comment]
It's a shame you can't get a status/tracking response via automation. Just so you know it has been dropped.

**Comment by RainVV** [comment]
您发给我的信件已经收到。谢谢!This is an automatic reply, confirming that your e-mail was received.Thank you！
